### PR TITLE
RUM-3531 Add `isVisible` to webview wireframe

### PR DESCRIPTION
**Pull request by maxep**
## Session Replay WebView

Report if a webview wireframe in memory is visible or not.

### CODE REVIEW

File: schemas/session-replay/mobile/webview-wireframe-schema.json [hunk 1]
```diff
@@ -21,6 +21,11 @@
           "type": "string",
           "description": "Unique Id of the slot containing this webview.",
           "readOnly": true
+        },
+        "hasFocus": {
```
Comment (xgouchet) [inline schemas/session-replay/mobile/webview-wireframe-schema.json]: > [!IMPORTANT]
> It feels that the notion of focus is different on mobile than it is on the Web. You could have two webviews on the same screen, both visible, but only one which has focus. I would advise to use `isVisible` instead, as it's more accurate to what happens on the mobile.

**Review by xgouchet — CHANGES_REQUESTED** [review]

File: schemas/session-replay/mobile/webview-wireframe-schema.json (hunk 1 above)
Comment (maxep) [reply schemas/session-replay/mobile/webview-wireframe-schema.json]: Changed 👍

**Review by mquentin — APPROVED** [review]
LGTM

**Review by xgouchet — APPROVED** [review]
Nice, thanks

**Pull request merged**